tests/bluetooth/audio/cap_commander/src/test_broadcast_reception.c

```diff
@@ -91,9 +113,9 @@ static void test_start_param_init(void *f)
 
 	fixture->start_param.count = ARRAY_SIZE(fixture->start_member_params);
 
-	for (size_t i = 0; i < ARRAY_SIZE(fixture->start_subgroups); i++) {
-		fixture->start_subgroups[i].bis_sync = i;
-		fixture->start_subgroups[i].metadata_len = 0;
+	for (size_t i = 0; i < ARRAY_SIZE(fixture->subgroups); i++) {
+		fixture->subgroups[i].bis_sync = 1 << i;
```

Comment:
Why are we shifting the bis_sync?

Reply:
ping @kruithofa

Reply:
To ensure that each subgroup has a different bis_sync